packages/api/sequelize.config.cjs

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  development: {
```

Comment:
I just checked and all our environments have been configured with `NODE_ENV` set to `production`. So this key needs to be named `production` for it to work in the environments. However it doesn't strike me as ideal that we have dev and test environments set as `production`! So for now, can this object also include `development` and `test` (with the same values). Then we're covered if in future we change `NODE_ENV` to more suitable values.